website/docs/docs/dbt-cloud-apis/project-state.md

```diff
@@ -61,16 +61,20 @@ Most Discovery API use cases will favor the _applied state_ since it pertains to
 
 | Node                                          | Executed in DAG  | Created by execution | Exists in database | Lineage               | States               |
 |-----------------------------------------------|------------------|----------------------|--------------------|-----------------------|----------------------|
-| [Model](/docs/build/models)                   | Yes              | Yes                  | Yes                | Upstream & downstream | Applied & definition |
-| [Source](/docs/build/sources)                 | Yes              | No                   | Yes                | Downstream            | Applied & definition |
-| [Seed](/docs/build/seeds)                     | Yes              | Yes                  | Yes                | Downstream            | Applied & definition |
-| [Snapshot](/docs/build/snapshots)             | Yes              | Yes                  | Yes                | Upstream & downstream | Applied & definition |
+| [Analysis](/docs/build/analyses)   	        | No               | No                   | No                 | N/A 	               | Definition 	      |
 | [Data test](/docs/build/data-tests)           | Yes              | Yes                  | No                 | Upstream              | Applied & definition |
 | [Exposure](/docs/build/exposures)             | No               | No                   | No                 | Upstream              | Definition           |
-| [Metric](/docs/build/metrics-overview)     | No               | No                   | No                 | Upstream & downstream | Definition           |
-| [Semantic model](/docs/build/semantic-models) | No               | No                   | No                 | Upstream & downstream | Definition           |
 | [Group](/docs/build/groups)                   | No               | No                   | No                 | Downstream            | Definition           |
 | [Macro](/docs/build/jinja-macros)             | Yes              | No                   | No                 | N/A                   | Definition           |
+| [Metric](/docs/build/metrics-overview)     	| No               | No                   | No                 | Upstream & downstream | Definition           |
+| [Model](/docs/build/models)                   | Yes              | Yes                  | Yes                | Upstream & downstream | Applied & definition |
+| [Saved query](/docs/build/saved-queries)      | Yes              | Yes                  | Yes                | Upstream & downstream | Definition 	      |
+| [Seed](/docs/build/seeds)                     | Yes              | Yes                  | Yes                | Downstream            | Applied & definition |
+| [Semantic model](/docs/build/semantic-models) | No               | No                   | No                 | Upstream & downstream | Definition           |
```

Comment:
Does the _Saved query_ lineage include "Downstream"?

Reply:
confirmed by eng that Saved Queries" are not currently exposed through the Discovery API and not exposed in upstream or downstream lineage yet

Reply:
Hiya @mirnawong1 

I'm probably reading this wrong I wanted to ask in the **Affected states by node type** table for Saved Queries - under the Lineage header it says **Upstream & downstream** with the info you found out from eng, should that read **N/A**, **not exposed in upstream or downstream lineage** etc.

Kind Regards